ui/src/app/views/workspace-edit.tsx

```diff
@@ -722,8 +773,27 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </div>
           </div>
         </WorkspaceEditSection>
-        <WorkspaceEditSection header='Request a review of your research purpose'
+        <WorkspaceEditSection header='Request a review of your research purpose for potential
```

Comment:
Note: the mocks have Research Purpose capitalized. I don't agree with that capitalization though (I'd keep it how you have it here)..

Reply:
karthik is fine with it lowercase